### PR TITLE
fix installer

### LINKED ISSUE #110: Installer is broken
**Describe the bug**
Not all changes were pushed when https://github.com/lightbend/cloudflow/pull/65, especially these part: https://github.com/lightbend/cloudflow/commit/6c78ce20b62f4988cea0a7ceeb6700d07b8af029
The initial issue is:
```./install.sh: line 75: unexpected EOF while looking for matching `"'
./install.sh: line 76: syntax error: unexpected end of file
```
I will crate a PR shortly.
**To Reproduce**
Running the installer on GKE fails.
**Expected behavior**
Installer should work.

**Screenshots**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by skonto**
### What changes were proposed in this pull request?
Fixes #110 

### Why are the changes needed?
Fix an urgent installer bug.

### Does this PR introduce any user-facing change?
Nope.

### How was this patch tested?
Manually by running the installer against GKE and EKS.

### CODE REVIEW

File: installer/create-cluster-eks.sh [hunk 1]
```diff
@@ -79,6 +79,12 @@ SECURITY_GROUP_IDS="$(aws eks describe-cluster --name "$CLUSTER_NAME" | jq -r '.
 echo "Security group id's: $SECURITY_GROUP_IDS"
 
 CLUSTER_SECURITY_GROUP_ID="$(aws eks describe-cluster --name "$CLUSTER_NAME" | jq -r '.cluster.resourcesVpcConfig.clusterSecurityGroupId')"
+CLUSTER_SECURITY_GROUP_ID_TEST_NULL="${CLUSTER_SECURITY_GROUP_ID/#null/}"
```
Comment (skonto) [inline installer/create-cluster-eks.sh]: We need this test as I get `null` when launching a cluster with my aws account.
